examples/deviceshifu/customizedshifu-humidity-detector-device/humidity-detector.go

```diff
@@ -0,0 +1,19 @@
+package main
```

Comment:
shall we have short folder names here and use snake style file name for consistency?
since we had examples/deviceshifu/customizedshifu-humidity-detector-device/humidity-detector.go
it looks examples/deviceshifu/customized/humidity_detector.go express the same purpose and looks much neat.